ui/src/index.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import moment from 'moment-timezone';
 import './index.css';
 import App from './components/App';
 
+// Set global timezone to UTC
+moment.tz.setDefault('UTC');
```

Comment:
why UTC

Reply:
Mostly this: https://zachholman.com/talk/utc-is-enough-for-everyone-right

I can have it append timezone information I guess but figured since we're really just interested in Date and not time we're better off having a consistent format for everyone using the application regardless of where they are logging in from.